easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.0-001-system.yml

```diff
@@ -0,0 +1,2 @@
+easyconfigs:
+  - Java-11.0.27.eb 
```

Comment:
@TopRichard the Java rebuilds have been ingested (see #1102), so this can be removed again.